paf-mvp-core-js/src/model/model.ts

```diff
@@ -20,3 +21,8 @@ export interface PAFNode {
   hostName: string;
   privateKey: string;
 }
+
+export interface RedirectErrorResponse {
```

Comment:
Documenting the interface would be valuable I think. Let me know if I am wrong but the properties of this interface are used to share more detail of the issue in the query parameters of the redirected URL, isn't it ? This is tacite with we see this interface and we don't connect the dots easily, I think.

Reply:
Ok